packaging/linux/snap/snapcraft.yaml

```diff
@@ -0,0 +1,24 @@
+name: keybase
+version: master
+summary: Crypto for everyone!
+description: |
+  Keybase service and command line client.
+
+grade: devel # must be 'stable' to release into candidate/stable channels
+confinement: strict
+
+apps:
+  keybase:
+    command: keybase
+    plugs: [network, network-bind]
+
+parts:
+  client-go:
+    source: ../../go
+    plugin: go
+    go-importpath: github.com/keybase/client/go
+    go-packages: [github.com/keybase/client/go/keybase]
+    go-buildtags: [production]
+    after: [go]
+  go:
+    source-tag: go1.7.5
```

Comment:
:laughing: good, your opinion will be very useful to move the discussion then.
Also, I forgot to mention that there's nothing stopping you to use your existing build script. You might like syncthing's approach: https://github.com/keybase/client/pull/6253#discussion_r110457959